src/styles/cytoscape/cytoscape-custom.css

```diff
@@ -5,6 +5,13 @@
   z-index: 10;
 }
 
+#cytoscape { 
+  width: 100%;
+  height: 800px;
+  background-color: #222;
+  z-index: 100;
```

Comment:
Even though you have z-index here, note that #183 happens already with the code on `master`

Reply:
I have a fix for this, it just involves z-indexes. I can create a pull request once this issue is resolved